pandas/core/window/indexers.py

```diff
@@ -120,3 +120,23 @@ def get_window_bounds(
             np.zeros(num_values, dtype=np.int64),
             np.arange(1, num_values + 1, dtype=np.int64),
         )
+
+
+class FixedForwardWindowIndexer(BaseIndexer):
+    """Calculate fixed forward-looking rolling window bounds,"""
+
+    @Appender(get_window_bounds_doc)
+    def get_window_bounds(
```

Comment:
is this a sufficient doc-string?

Reply:
Expanded and clarified the class docstring a bit and added examples directly to it. I think now it's clear just what the new class does.
The `get_window_bounds` docstring is the same for every `BaseIndexer` subclass. We could add a particular docstring to the function instead of using the Appender, but I don't think it would add much clarity for the user.